wp-content/themes/workingnyc/views/components/utility-nav.twig

```diff
@@ -1,10 +1,10 @@
 <div class="c-utility wrap">
-  <a class="link-icon mie-auto" href="{{ return.link }}">
```

Comment:
utility-nav.twig is a shared component that is also used by jobs/single.twig and programs/single.twig. A change in this component would also affect how jobs and programs are displayed

- [ ] Revert this change